src/main/java/tectech/thing/metaTileEntity/multi/godforge/MTEPlasmaModule.java

```diff
@@ -92,12 +92,12 @@ protected CheckRecipeResult validateRecipe(@NotNull GTRecipe recipe) {
             @Override
             protected CheckRecipeResult onRecipeStart(@NotNull GTRecipe recipe) {
                 wirelessEUt = (long) recipe.mEUt * maxParallel;
-                if (!addEUToGlobalEnergyMap(userUUID, -calculatedEut * duration)) {
+                BigInteger powerForRecipe = BigInteger.valueOf(calculatedEut)
+                    .multiply(BigInteger.valueOf(duration));
```

Comment:
Please change this line so it accounts for parallels as well (should be more accurate that way)
```suggestion
                    .multiply(BigInteger.valueOf((long) duration * getActualParallel())));
```

Reply:
calculatedEut already contains parallel here

Reply:
Looks like you are correct